Documentation/specs/runtime-configuration-file.md

```diff
@@ -128,9 +123,13 @@ This section is copied verbatim from an identical section in the input `project.
 
 These settings are read by `corehost` to determine how to initialize the runtime. All versions of `corehost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
 
+### `targetRuntime` Section (`.runtimeconfig.json`)
+
+This section indicates the name and version of the shared runtime in use. The `dotnet publish` command will place this data here (where it gets that data from is undecided but out of the scope of this document). If not present, the host will produce an error.
+
```

Comment:
Why would the host produce an error if this isn't present? I would expect this to not be present in a standalone application for example.